.jshintrc

```diff
@@ -16,6 +16,9 @@
     "node": true, // Enable globals available when code is running inside of the NodeJS runtime environment.
     "mocha": true,
 
+    // Relaxing options
+    "boss": true, // Accept statements like `while (key = keys.pop()) {}`
```

Comment:
I personally like to use this. I could imagine though that not everyone likes it?